.github/workflows/package_and_push_helm_chart.yaml

```diff
@@ -0,0 +1,53 @@
+name: Package and Push Helm Chart
+
+on:
+  push:
+    branches:
+      - dev
+      - release
+  workflow_dispatch:
+    inputs:
+      name:
+        description: "Manual trigger for Helm chart packaging and pushing"
+      release_type:
+        description: "Release or snapshot version?"
+        required: true
+        default: "SNAPSHOT"
+        options:
+          - "SNAPSHOT"
+          - "RELEASE"
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Setup Helm
+        uses: azure/setup-helm@v4.1.0
+        with:
+          version: ${{ vars.HELM_VERSION }}
+
+      - name: Login with Helm
+        run: |
+          echo ${{ secrets.GITHUB_TOKEN }} | helm registry login ${{ vars.HELM_REGISTRY }} --username ${{ github.repository_owner }} --password-stdin
+
+      - name: Prepare version
+        run: |
+          VERSION=$(cat ./VERSION)
+          if [[ ${{ github.event.inputs.environment }} = "SNAPSHOT" ]] || [[ $BRANCH = "dev" ]]; then
+            VERSION=$VERSION-SNAPSHOT
+          fi
+          echo "VERSION=$VERSION" >> $GITHUB_ENV
+
+      - name: Push Helm chart to repository
+        run: |
+          helm package "charts/fairspace" --version ${{ env.VERSION }}
```

Comment:
before that it would be useful to run `helm dependency update fairspace` (as it is in the local deployment script: https://github.com/thehyve/fairspace/blob/dev/local-development/fairspace/deploy.sh#L48)

Reply:
we don't have dependencies, no need for now, but it is better to have in case one day I will have some - added the command, thanks for the point!

Reply:
on a really small note - not sure if the quotes are needed for `charts/fairspace`, but now there are no quotes for `dependency update` and they are there for `helm package`. Maybe good to make it consistent?

Reply:
no problem, added